examples/pytorch/MNIST/README.md

```diff
@@ -1,3 +1,82 @@
-## MNIST Handwritten Digit Recognition examples
+## MNIST example with MLFlow
+In this example, we train a Pytorch Lightning model to predict handwritten digits, leveraging early stopping.
+The code, adapted from this [repository](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pl_examples/basic_examples/mnist.py), is almost entirely dedicated to model training, with the addition of a single ``mlflow.pytorch.autolog()`` call to enable automatic logging of params, metrics, and models,
+including the best model from early stopping.
+
+### Running the code
+To run the example via MLflow, navigate to the `mlflow/examples/pytorch/MNIST/example1` directory and run the command
+
+```
+mlflow run .
+```
+
+This will run `mnist_autolog_example1.py` with the default set of parameters such as  `--max_epochs=5`. You can see the default value in the `MLproject` file.
```

Comment:
```suggestion
This will run `mnist_autolog_example.py` with the default set of parameters such as  `--max_epochs=5`. You can see the default value in the `MLproject` file.
```

Can we rename the script to just `mnist_autolog_example.py` now that there's only one script in the `MNIST` directory?

Reply:
renamed it to `mnist_autolog_example.py`